aws/aws-parallelcluster-cookbook
# Add get-compute-fleet-status and update-compute-fleet-status script

### Description of changes
* Add `get-compute-fleet-status.sh` and  `update-compute-fleet-status.sh` under /usr/local/bin

### Tests
* Manually tests:
```
[ec2-user@ip-10-0-0-137 bin]$ get-compute-fleet-status.sh -h

  Usage: get-compute-fleet-status.sh [OPTION]...

  Copy the AWS ParallelCluster Package to an S3 bucket.

  -h, --help                    Print this help message
```
```
[ec2-user@ip-10-0-0-137 bin]$ get-compute-fleet-status.sh
{
    "lastStatusUpdatedTime": "2022-01-19 01:25:56 UTC",
    "status": "RUNNING"
}
echo $?
0
```

```
[ec2-user@ip-10-0-0-137 bin]$ update-compute-fleet-status.sh -h

  Usage: update-compute-fleet-status.sh [OPTION]...

  Copy the AWS ParallelCluster Package to an S3 bucket.

  --status                      Expected status of compute fleet
  -h, --help                    Print this help message

```
```
[ec2-user@ip-10-0-0-137 bin]$ update-compute-fleet-status.sh --status PROTECTED
echo $?
0

[ec2-user@ip-10-0-0-137 bin]$ get-compute-fleet-status.sh
{
    "lastStatusUpdatedTime": "2022-01-19 01:34:10.177271+00:00",
    "status": "PROTECTED"
}
```
```
[ec2-user@ip-10-0-0-137 bin]$ update-compute-fleet-status.sh --status RUNNING
usage: compute_fleet_status.py [-h] --table-name TABLE_NAME --region REGION
                               [--status {PROTECTED}] --action {get,update}
compute_fleet_status.py: error: argument --status: invalid choice: 'RUNNING' (choose from 'PROTECTED')
echo $?
1
```
```
[ec2-user@ip-10-0-0-137 bin]$ update-compute-fleet-status.sh
ERROR: --status parameter not specified
echo $?
1

```
error are redirected to stderr
```
for $ update-compute-fleet-status.sh 2>/dev/null output is empty
for $ update-compute-fleet-status.sh --status RUNNING 2>/dev/null output is empty
```
